src/solvers/minimize.jl

```diff
@@ -1,37 +1,49 @@
-export @minimize 
+export @minimize
 
 """
-`@minimize cost [st ctr] [with slv_opt]`
+    @minimize cost [st ctr] [with slv_opt]
 
-Minimize a given problem with cost function `cost`, constraints `ctr` and solver options `slv_opt`. 
+Minimize a given problem with cost function `cost`, constraints `ctr` and solver options `slv_opt`.
 
 # Example
 
 ```julia
-julia> A, b = randn(10,4), randn(10);
+julia> using StructuredOptimization, ProximalAlgorithms
+
+julia> A, b, x = randn(10,4), randn(10), Variable(4);
 
 julia> @minimize ls(A*x-b) + 0.5*norm(x);
-    it |      gamma |        fpr |        tau |        FBE |
- ------|------------|------------|------------|------------|
-     1 | 2.9152e-02 | 2.7656e+00 | 1.0000e+00 | 5.5181e+00 |
-     9 | 2.9152e-02 | 9.9682e-05 | 1.0000e+00 | 4.4086e+00 |
+
+julia> ~x
+4-element Array{Float64,1}:
+ -0.3606117955906451
+  0.21231949195945288
+  0.022423882830584645
+  0.19560804140788765
 
 julia> @minimize ls(A*x-b) st x >= 0.;
-    it |      gamma |        fpr |        tau |        FBE |
- ------|------------|------------|------------|------------|
-     1 | 5.8304e-02 | 1.0068e+00 | 1.0000e+00 | 6.6282e+00 |
-     3 | 5.8304e-02 | 9.5210e-16 | 1.0000e+00 | 6.5654e+00 |
 
-julia> it, slv = @minimize ls(A*x-b) st norm(x) == 2.0 with PG(maxit = 5);
-    it |      gamma |        fpr |
- ------|------------|------------|
-     1 | 6.1373e-02 | 2.2090e+01 |
-     5 | 3.0686e-02 | 5.5190e-01 |
+julia> ~x
+4-element Array{Float64,1}:
+ 0.0
+ 0.2682589660003247
+ 0.0
+ 0.2297141914608737
 
-```
+julia> @minimize ls(A*x-b) st norm(x) == 2.0 with ProximalAlgorithms.ForwardBackward(fast=true);
```

Comment:
This is too long in my opinion

Reply:
I've simplified it a bit: we will further simplify it if we do the "re-export" thing